src/System.Security.Cryptography.Encoding/src/System/Security/Cryptography/AsnEncodedDataCollection.cs

```diff
@@ -85,7 +85,16 @@ void ICollection.CopyTo(Array array, int index)
 
         public void CopyTo(AsnEncodedData[] array, int index)
         {
-            ((ICollection)this).CopyTo(array, index);
+            // Need to do the argument validation ourselves as AsnEncodedDataCollection throws ArgumentOutOfRangeException where List<>.CopyTo() throws
```

Comment:
The fact that this comment is the only reason this file gets a horizontal scrollbar on github UI makes me a little sad; but it's not something I'll fight over.